.github/ISSUE_TEMPLATE/NewLibraryRequest.md

```diff
@@ -5,6 +5,8 @@ labels: Weekly, AutoAssignerAppLibraryReview
 ---
 In order to properly evaluate if a new library can be added to `package.json`, please fill out this request form. It will be automatically assigned someone from our review team that will go through and vet the library.
 
+*In order to add any new production dependency, it must receive a :+1: from at least 51% of the app deployers.*
```

Comment:
My thought was kind of different than this...

> I kinda feel like the vote is more like a WN vote. The main benefit is for getting feedback, and the vote itself is less important.

I don't feel that strongly either way, but I will say that it can be difficult to get a 51% majority vote sometimes.

Reply:
And I mean that in the sense that it's hard to get that many people's attentions to vote.

Reply:
I hear ya, and FWIW I don't feel that strongly either. an advantage of a strict threshold for the vote is that it makes it a very clear pass/fail.

For context, this was spurred by [this comment](https://expensify.slack.com/archives/C01GTK53T8Q/p1708591427090919?thread_ts=1707467001.447909&cid=C01GTK53T8Q)

Reply:
OK, cool. So that we can give contributors some clarity, how about saying it like this:

> In order to add any new production dependency, it must be approved by the App Deployer team. They will evaluate the library and decide if it's something we want to move forward with or if other alternatives should be explored.